src/js/lib/jquery.kazoosdk.js

```diff
@@ -780,6 +787,14 @@
 			var postData = data.data,
 				envelopeKeys = {};
 
+			// Remove object's metadata (if any) merged in the success step.
+			if (postData.hasOwnProperty('metadata')) {
+				_.each(postData.metadata, function(index, name) {
+					name !== 'id' && delete postData[name];
+				});
+				delete postData.metadata;
```

Comment:
Similarly here, we can use jQuery's `each()` instead:
```suggestion
				$.each(postData.metadata, function(key) {
					if (key === 'id') {
						return;
					}
					delete postData[key];
				});
```